File: fix_seo.js
const fs = require('fs');
let content = fs.readFileSync('seo.js', 'utf-8');

content = content.replace(/openSeoWork\('\$\{proj\.id\}'/g, "openSeoWork('${proj.assignment_id || proj.id}'");
content = content.replace(/openProjectSheetAccess\('\$\{proj\.id\}'/g, "openProjectSheetAccess('${proj.project_id || proj.id}'");
content = content.replace(/reactivateSeo\('\$\{proj\.id\}'\)/g, "reactivateSeo('${proj.assignment_id || proj.id}')");
content = content.replace(/submitGmbReportDirectly\('\$\{proj\.id\}'\)/g, "submitGmbReportDirectly('${proj.assignment_id || proj.id}')");
content = content.replace(/extendGmbPostScheduleFromCard\('\$\{proj\.id\}'/g, "extendGmbPostScheduleFromCard('${proj.assignment_id || proj.id}'");

fs.writeFileSync('seo.js', content, 'utf-8');
console.log("Replaced successfully!");

File: fix_seo.py
import re

with open('seo.js', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace  with 
# Specifically in the onclick handlers
content = content.replace("openSeoWork(''", "openSeoWork(''")
content = content.replace("openProjectSheetAccess(''", "openProjectSheetAccess(''")
content = content.replace("reactivateSeo('')", "reactivateSeo('')")
content = content.replace("submitGmbReportDirectly('')", "submitGmbReportDirectly('')")
content = content.replace("extendGmbPostScheduleFromCard(''", "extendGmbPostScheduleFromCard(''")


with open('seo.js', 'w', encoding='utf-8') as f:
    f.write(content)
print("Updated seo.js proj.id references!")
